.ci/cico_cleanup_dev_cluster.sh

```diff
@@ -12,6 +12,7 @@
 export DEV_CLUSTER_URL=https://devtools-dev.ext.devshift.net:8443/
 
 source ./.ci/functional_tests_utils.sh
+installEpelRelease
```

Comment:
could you clarify why do we need this call ?

Reply:
Epel is needed to be installed because it provides JQ which is installed right after.

Reply:
maybe the `installJQ` function can take care of that? But that's just a minor note, not preventing the merge from my POV.

Reply:
@Katka92 agree with @rhopp `installEpelRelease` does not really needed if the only reason it is required is for JQ. Please, consider refactoring since installing Epel is just an impl detail in this case

Reply:
I've moved enabling of epel. @ibuziuk could you please review again?